indico/modules/auth/client/js/signup.jsx

```diff
@@ -291,7 +291,7 @@ function Signup({
             </Fieldset>
           )}
           {termsRequireAccept && !!privacyPolicy && !privacyPolicyUrl && (
-            <Fieldset legend={Translate.string('Privacy policy')}>
+            <Fieldset legend={Translate.string('Privacy Policy')}>
```

Comment:
Are we sure we want to change the casing? I think we'd lose a translation

Reply:
We won't because both versions exist and that's also why I wanted to make the spelling more consistent :)